src/supabase.ts

```diff
@@ -0,0 +1,41 @@
+const commonOptions: Fig.Option[] = [
+  {
+    name: ["--help", "-h"],
+    description: "Show help for command",
+    priority: 1,
+  },
+];
+
+const completionSpec: Fig.Spec = {
+  name: "supabase",
+  description:
+    "The CLI is still under heavy development, but it will contain all the functionality for working with Supabase projects and the Supabase platform",
+  subcommands: [
+    {
+      name: "init",
+      description:
+        "This command will create a .supabase folder which holds all the configuration for developing your project locally. You don't need to check this into version control",
+      options: [...commonOptions],
```

Comment:
nit: a new array isn't necessary here. Same for the other occurrences in the file
```suggestion
      options: commonOptions,
```